Snabble/UI/Scanner/ShoppingListCell.swift

```diff
@@ -107,17 +90,17 @@ extension ShoppingListCell {
         }
 
         if let img = ShoppingListCell.imageCache[imgUrl] {
-            self.productImage.image = img
+            self.cellView?.configureProductImage(with: img)
             return
         }
 
-        self.spinner.startAnimating()
-        self.spinner.isHidden = false
+        self.cellView?.spinnerStartAnimation()
+        self.cellView?.spinnerIsHidden(false)
         self.task = Snabble.urlSession.dataTask(with: url) { data, _, error in
             self.task = nil
             DispatchQueue.main.async {
-                self.spinner.stopAnimating()
-                self.spinner.isHidden = true
+                self.cellView?.spinnerStopAnimation()
+                self.cellView?.spinnerIsHidden(true)
```

Comment:
I would recommend to use [`hidesWhenStopped`](https://developer.apple.com/documentation/uikit/uiactivityindicatorview/1622837-hideswhenstopped#) instead of `spinnerIsHidden(_:)`.